packages/api/test/auth.test.ts

```diff
@@ -18,13 +17,14 @@ const user = 'username';
 const pass = 'changeme';
 
 describe('#auth()', function () {
-  this.beforeAll(function () {
+  before(function () {
     // Set a unique cache dir so these tests won't collide with other tests and we don't need to go
     // through the trouble of mocking out the filesystem.
     Cache.setCacheDir(uniqueTempDir());
   });
 
-  beforeEach(function () {
+  beforeEach(async function () {
+    const securityOas = await loadSpec('@readme/oas-examples/3.0/json/security.json');
```

Comment:
I changed up all of these spec imports because `oas.dereference` has the tendency to sometimes update the reference of the spec that it's loading, causing these imports sometimes to import a the content of an already dereferenced spec from another test -- and then it itself dereferencing again causing all sorts of flakiness. `loadSpec` loads the spec and then clones the object with `JSON.stringify` and `JSON.parse`.